resources/patches/ruby-stomp/verify_client_certs.patch

```diff
@@ -1,69 +0,0 @@
-From d18bb48b93d0af2ebdb936811fb980706dfcecf8 Mon Sep 17 00:00:00 2001
```

Comment:
Present at https://github.com/stompgem/stomp/commit/4b4d09009b6011d46f948600cdb935980243fbe5 and  https://github.com/stompgem/stomp/commit/3b9fb734a6508de9f7c9769894cc3a8a0ad8b252.